nvi-commons/src/main/java/no/sikt/nva/nvi/common/utils/DynamoDbUtils.java

```diff
@@ -72,8 +72,15 @@ private static AttributeValue mapToDynamoDbValue(
     if (containsAttributeValueMap(value)) {
       return mapEachAttributeValueToDynamoDbValue(value);
     }
-    if (nonNull(value.getL()) && value.getL().isEmpty()) {
-      return AttributeValue.builder().l(emptyList()).build();
+    if (nonNull(value.getL())) {
+      if (value.getL().isEmpty()) {
+        return AttributeValue.builder().l(emptyList()).build();
+      }
+      var converted =
+          value.getL().stream()
+              .map(item -> attempt(() -> mapToDynamoDbValue(item)).orElseThrow())
```

Comment:
Maybe throw a less generic exception with a message, unless this is a true "will never happen"

Reply:
It's really the same `JsonProcessingException` we already have in the signature and which we handle with `orElseThrow` for other cases. If we change it here, we should change it there as well. Currently I think it bubbles up to this error handling:


```
private Dao extractDao(DynamodbStreamRecord streamRecord) throws Exception {
    var image = getImage(streamRecord);
    return attempt(() -> DynamoEntryWithRangeKey.parseAttributeValuesMap(image, Dao.class))
        .orElseThrow(
            daoFailure -> {
              LOGGER.error("Failed to parse image: {}", image.toString());
              return daoFailure.getException();
            });
  }
 ```